include/custom_functions.h

```diff
@@ -155,7 +160,10 @@ uint32_t set_fixed_num(const char * s, int num_fractional_bits, bool is_signed)
   }
   accumulator += int_part_decimal << num_fractional_bits;
   if (is_signed && s[0] == '-') {
-    accumulator |= 0x80000000; // if its a signed int and theres a negative sign flip the first bit of accumulator
+    // Wrong this is just doing sign-magnitude and we need 2's complement
```

Comment:
Let's change this comment since it won't make sense without the old, incorrect, sign-magnitude code.

Reply:
Good catch. That was a comment to myself since I found the issue and then went on lunch.